packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1332,6 +1426,11 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
 +    # Change this if you change the namespaceOverride or nameOverride of prometheus-operator
 +    url: http://rancher-monitoring-prometheus.cattle-monitoring-system.svc
 +    port: 9090
++  image:
++    repository: directxman12/k8s-prometheus-adapter-amd64
```

Comment:
and this one is going to be updated in the next pr that you mentioned in the issue?

Reply:
Yeah, this will be updated in the next PR based on the [comment](https://github.com/rancher/rancher/issues/28441#issuecomment-678939966) on the issue as you mentioned!

Reply:
this will later be updated to `rancher/directxman12-k8s-prometheus-adapter-amd64` ?

Reply:
Yes, in this PR: https://github.com/rancher/charts/pull/582/commits/7752a3978ad3d86b5eaf17ca10081e520e087675#diff-ed5b5b87cf149adc6a6408bf81993efdR1703